server/src/test/java/com/lunchtime/stub/PersonServiceStub.java

```diff
@@ -0,0 +1,65 @@
+package com.lunchtime.stub;
+
+import com.lunchtime.models.Person;
+import com.lunchtime.service.PersonService;
+import com.lunchtime.service.dto.PersonDto;
+import com.lunchtime.service.dto.PersonPassDto;
+import com.lunchtime.service.dto.RegisterPerson;
+
+import java.util.Optional;
+
+public class PersonServiceStub implements PersonService {
```

Comment:
Why you create this class, but don't use it?

Reply:
File deleted